config.go

```diff
@@ -62,6 +62,14 @@ type DatabaseConfig struct {
 	Params    map[string]string
 	TLS       *TLSConfig
 
+	// ReadTimeout is used to configure the MySQL client timeout for waiting for data from server.
+	// Timeout is in seconds. Defaults to unlimited.
+	ReadTimeout uint64
+
+	// WriteTimeout is used to configure the MySQL client timeout for writing data to server.
+	// Timeout is in seconds. Defaults to unlimited.
```

Comment:
Minor nit: maybe  "defaults to 0, which means no timeout (see https://github.com/go-sql-driver/mysql#timeout)"?